docs: added information for developers with cross-platform recomendat… (#88)
Haulmont/jmix-frontend: CONTRIBUTING.md

> NOTE: This section is under construction.

#### Cross-platforming

> When you working with code, you need to adhere to several recomendations for achive correct app working on UNIX and Windows OS:

Reviewer: ```suggestion
When you are working with code, you need to adhere to several recommendations in order for the app to work correctly on both UNIX and Windows OS:
```

> 1. Remember, that Windows uses "\\" symbol as path separator, but UNIX uses "/". For this reason, you need to use "path" package, when you working with pathes in code:

Reviewer: ```suggestion
1. Remember that Windows uses "\" symbol as path separator, but UNIX uses "/". For this reason, you need to use "path" package when you work with paths in code:
```

Author: @vyacheslav-pushkin "\\" - this is done on purpose to show \ symbol in ""

```
Examples:

// wrong
const examplePath = 'my/test/directory';

// correct
const examplePath = path.join('my', 'test', 'directory');
``` 

> 2. Avoid using CLI commands which dependend from OS, like 'mkdir -p', 'rm -rf' e.t.c. Instead, use npm packages wiht CLI which have the same functionality:

Reviewer: ```suggestion
2. Avoid using CLI commands which depend on OS, like 'mkdir -p', 'rm -rf', etc. Instead, use npm packages with CLI that have the same functionality:
```

```
Examples (package.json scripts):

// wrong
"clean": "rm -rf dist && rm -rf dist-transpiled",
"dist": "npm run compile && mkdir -p dist-browser && browserify --standalone cuba dist-node/cuba.js > dist-browser/cuba.js"

// correct
"clean": "rimraf dist && rimraf dist-transpiled",
"dist": "npm run compile && mkdirp dist-browser && browserify --standalone cuba dist-node/cuba.js > dist-browser/cuba.js"
``` 

> 3. When you working with shell scripts, you need to provide .sh script for UNIX users and .bat for Windows users. Also, you need to implement logic of runing .sh scripts for UNIX users, and .bat for Windows users.

Reviewer: ```suggestion
3. When you are working with shell scripts, you need to provide `.sh` script for UNIX users and `.bat` for Windows users. Also, you need to implement logic of running `.sh` scripts for UNIX users, and `.bat` for Windows users.
```
